gpii/node_modules/transformer/test/js/TransformerTests.js

```diff
@@ -155,9 +152,7 @@ fluid.setLogging(true);
                         "encoding": "utf-8",
                         "xml-tag": "<?xml version=\"1.0\"?>"
                     },
-                    "capabilities": [
-                        "applications.eu\\.singularlogic\\.pixelsense\\.sociable.id"
-                    ],
+                    "capabilities": [],
```

Comment:
The <application-id>.id ones are now assumed (akwardly done here: https://github.com/amb26/universal/pull/7/files#diff-f74832f415cec8b5a9b062df2276d1acR133), so I've removed all those entries from the solutions' capabilities.  

 Other capabilities are still valid.